kube/src/config/utils.rs

```diff
@@ -75,6 +77,47 @@ pub fn is_expired(timestamp: &str) -> bool {
     ts < now
 }
 
+// Copied from rustls::internal::pemfile::certificates
```

Comment:
shame it's so hard to find utility libraries/helpers for these things, remember having to dig into rustls::internal::pemfile about a year ago :|

Reply:
Yeah :| I needed this on both `native-tls` and `rustls-tls`.

Reply:
Well AFAIK `rustls::internal::pemfile::certs` is still public, although unstable? Seems like it'd be better to have explicit breakage if this has to change than sticking to the "wrong" behaviour.

Reply:
I'm using this from both `native-tls` and `rustls-tls` so that `Config`'s `root_cert` is always `Option<Vec<Der>>` (was `Option<Vec<reqwest::Certificate>>`). And I had to copy because I can't use `rustls` with feature `native-tls`.

Reply:
I found this can be implemented using `pem::parse_many` (this one uses `regex` though) and filtering it, so I've removed the copied code.